src/DotLiquid/Strainer.cs

```diff
@@ -87,15 +57,14 @@ public Strainer(Context context)
             _context = context;
         }
 
-
         /// <summary>
         /// In this C# implementation, we can't use mixins. So we grab all the static
         /// methods from the specified type and use them instead.
         /// </summary>
         /// <param name="type"></param>
         public void Extend(Type type)
         {
-            // From what I can tell, calls to Extend should replace existing filters with the same number of params. So be it.
+            // Calls to Extend should replace existing filters with the same number of params.

```

Comment:
nit: remove "should" for the same reason.

Reply:
updated: https://github.com/dotliquid/dotliquid/pull/453/commits/faa0f59f2b6bd758fd666140edd2d75842a9004c